client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -142,7 +142,7 @@ class StatsPostDetail extends Component {
 		const isFeatured = config.isEnabled( 'stats/enhance-post-detail' );
 
 		return isFeatured ? (
-			<Main fullWidthLayout>
+			<Main fullWidthLayout className="stats-post-detail">
```

Comment:
```suggestion
			<Main fullWidthLayout>
```
Could we just add class names to specific pages for the `.highlight-cards` [background color](https://github.com/Automattic/wp-calypso/pull/72756/files#diff-495d35474fbde4de50d929b36664a994f4235e25e86b9f3b1da7941ba16da404R64-R68)?

Reply:
Addressed in ec1a68007c